api/ngrok/v1alpha1/cloudendpoint_types.go

```diff
@@ -63,13 +80,16 @@ type CloudEndpointStatus struct {
 	ID string `json:"id,omitempty"`
 }
 
+// CloudEndpoint is the Schema for the cloudendpoints API
+//
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+// +kubebuilder:resource:categories="networking";"ngrok"
+// +kubebuilder:resource:shortName=cep
```

Comment:
Very nice on the shortName. I think internally we use the shorthand `clep` for cloud endpoints. Might want to get products thoughts on what this should be and make it consistent across our products.